rules/combo/stealer/browser.yara

```diff
@@ -21,26 +21,39 @@ rule multiple_browser_credentials : high {
     3 of ($c_*) and none of ($not_*)
 }
 
-rule multiple_browser_credentials_2 {
+rule multiple_browser_refs : critical {
+  meta:
+	description = "Makes references to multiple browser credentials"
   strings:
-    $a_google_chrome = "Google/Chrome"
-    $a_app_support = "Application Support"
-    $a_app_support_slash = "Application\\ Support"
-    $a_cookies_sqlite = "cookies.sqlite"
-    $a_cookies = "Cookies"
-    $a_places_sqlite = "places.sqlite"
-    $a_moz_cookies = "moz_cookies"
-    $a_firefox_profiles = "Firefox/Profiles"
-    $a_opera_gx = "OperaGX"
-    $a_form_history = "formhistory.sqlite"
-    $a_chrome_local_state = "Chrome/Local State"
-    $a_brave_software = "BraveSoftware"
-    $a_opera = "Opera Software"
+    $fs_config = ".config" fullword
+    $fs_app_support = "Application Support"
+    $fs_app_support_slash = "Application\\ Support"
+    $fs_chrome = "Google/Chrome"
+    $fs_chrome_local_state = "Chrome/Local State"
+    $fs_chrome_userdata = "Chrome/User Data"
+    $fs_cookies = "Cookies"
+    $fs_cookies_sqlite = "cookies.sqlite"
+	$fs_firefox = "Mozilla/Firefox"
+    $fs_firefox_profiles = "Firefox/Profiles"
+    $fs_form_history = "formhistory.sqlite"
+    $fs_moz_cookies = "moz_cookies"
+    $fs_places_sqlite = "places.sqlite"
+	$fs_roaming = "Roaming/"
+	$fs_user_data = "User Data"
+
+	$name_brave = "Brave-Browser"
+    $name_brave_software = "BraveSoftware"
+    $name_chrome = "Google Chrome"
+    $name_opera_gx = "OperaGX"
+	$name_firefox = "Firefox"
+	$name_opera_gx_dir = "Opera GX"
+    $name_opera = "Opera Software"
+	$name_yandex = "YandexBrowser"
```

Comment:
Would it make sense to add Safari or Vivaldi here (DDG and Orion are also possibilities)?

Reply:
Safari is special as it cookies in `~/Library/Cookies` (covered by $f_cookies)

I haven't seen malware samples that target the others, but if I do, we'll add them here.